assets/js/googlesitekit/widgets/register-defaults.js

```diff
@@ -227,7 +227,7 @@ export function registerDefaults( widgetsAPI ) {
 					'google-site-kit'
 				),
 				subtitle: __(
-					'Track your sites traffic over time',
+					"Track your site's traffic over time",
```

Comment:
We use the `’` character for apostrophes in our code base.
```suggestion
					'Track your site’s traffic over time',
```